### PR TITLE
Revert VC140_ON_VS2017

### PR DESCRIPTION
**Pull request by isuruf**
Setting CC and CXX and using NMake/Ninja generator works with azure on conda-forge. I'm reverting VC140_ON_VS2017 because it didn't work as expected. (It should have worked, but there's something wrong with the logic and it's hard for me to debug without a windows setup.)

cc @mariusvniekerk

### CODE REVIEW

**Comment by isuruf** [comment]
ping @msarahan

**Comment by msarahan** [comment]
These packages never made it out of c3i_test2, I believe.  I believe no further builds are necessary.  Please correct me if I'm wrong.

File: vs2015/activate.bat [hunk 1]
```diff
@@ -73,6 +43,10 @@ set "CFLAGS=%CFLAGS% -MD -GL"
 set "CXXFLAGS=%CXXFLAGS% -MD -GL"
 set "LDFLAGS_SHARED=%LDFLAGS_SHARED% -LTCG ucrt.lib"
 
+:: set CC and CXX for cmake
+set "CXX=cl.exe"
+set "CC=cl.exe"
```
Comment (isuruf) [inline vs2015/activate.bat]: @msarahan, need a rebuild for this small change.

Comment (isuruf) [reply vs2015/activate.bat]: Maybe rebuild after merging https://github.com/AnacondaRecipes/aggregate/pull/138 ?

**Pull request merged**